config/config.go

```diff
@@ -183,3 +187,19 @@ func Load(ctx *cli.Context, loadNetworkConfig bool) (*Config, error) {
 	}
 	return cfg, nil
 }
+
+// NewKeyFromKeystore creates a private key from a keystore file
+func NewKeyFromKeystore(cfg types.KeystoreFileConfig) (*ecdsa.PrivateKey, error) {
```

Comment:
We could also already apply the related changes from Arnau here. [See here.](https://github.com/0xPolygon/cdk-validium-node/blob/a03bdeb526891222e22e138d2a83408b43559c25/etherman/etherman.go#L1739)